hpx/runtime/threads/detail/scheduled_thread_pool.hpp

```diff
@@ -362,6 +367,15 @@ namespace hpx { namespace threads { namespace detail
         std::vector<std::uint64_t> exec_times_, tfunc_times_;
         std::vector<std::uint64_t> reset_tfunc_times_;
 
+        //background duration for adaptive coalescing counters
+        std::vector<std::uint64_t> background_duration_;
```

Comment:
Shouldn't this be inside the pp guards below as well?

Reply:
I needed to exclude it from the guard as I am using that variable to create an object of type detail::scheduling_counters (https://github.com/justwagle/hpx/blob/9eac86a642d4e43cc5b359f7d8444170f166cda8/hpx/runtime/threads/detail/scheduled_thread_pool_impl.hpp#L576). If I included it in the guard, I got an error stating that background_duration_ was undefined when the counter was disabled.